tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Features/Reset Password.3926.feature

```diff
@@ -0,0 +1,40 @@
+﻿Feature: Reset Password
+	As a User 
+    I want to be able to reset my password
```

Comment:
Indentation

Reply:
Think this is an issue looking it on a browser, the indentation is fine on my version in VS

Reply:
It's caused by a mix of tabs and spaces – it should be one of all, spaces if I remember correctly @CallumBradburyBJSS?

Reply:
Yeah, all spaces, all the time. Visual studio should do that by default, but somehow it's being subverted somehow

Reply:
I suspect we'll find tab city if we check all our existing feature files so it might be worth a tech debt task.